codec-http2/src/test/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowControllerTest.java

```diff
@@ -1050,6 +1050,10 @@ public void error(Throwable t) {
         }
 
         @Override
+        public void writeComplete() {
```

Comment:
@buchgr Yeah, I think the code snippet in https://github.com/netty/netty/pull/3456#issuecomment-77281378 should address these